CMakeLists.txt

```diff
@@ -192,6 +193,11 @@ if(OGS_USE_EIGEN)
     endif()
 endif()
 
+if(MSVC AND OGS_32_BIT)
+    add_definitions(-DEIGEN_MAX_ALIGN_BYTES=0)
```

Comment:
Maybe 3.2 doesn't understand this yet:
http://eigen.tuxfamily.org/dox-3.2/TopicPreprocessorDirectives.html
Please try EIGEN_DONT_ALIGN